lib/file-store.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+const fsPromises = require('fs').promises;
+const uuidv4 = require('uuid/v4');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+
+const sqldb = require('@prairielearn/prairielib/sql-db');
+const config = require('../lib/config');
+
+module.exports = {
+
+    /**
+     * Upload a file into the file store.
+     *
+     * @param {string} display_filename - The display_filename of the file.
+     * @param {buffer} contents - The file contents.
```

Comment:
Capital-B `Buffer` for typechecking in the future 😉

Reply:
done